lib/main.dart

```diff
@@ -48,3 +51,41 @@ Future<void> main() async {
 
   runApp(OcneraApp());
 }
+
+Future<void> configureLogger() async {
+  Fimber.plantTree(DebugTree());
+  Directory extDir = await getApplicationDocumentsDirectory();
+  String path = '${extDir.path}/${GlobalConfiguration().getValue('LOG_DIR')}';
+  logger.i("Generating folder for local images $path");
+  await Directory(path).create(recursive: true);
+  deletePreviousFiles(path,
+      fileCount: GlobalConfiguration().getValue("LOG_HISTORY_FILES"));
+  var fileTree = TimedRollingFileTree(
+      timeSpan: TimedRollingFileTree.dailyTime,
+      filenamePrefix: '$path/appLog_',
+      filenamePostfix: '.log',
+      logLevels: ['I', 'W', 'E']);
+
+  Fimber.plantTree(fileTree);
+}
+
+void deletePreviousFiles(String path, {int fileCount = 3}) {
+  var logDir = Directory(path);
+  var files = logDir.listSync();
+  files.forEach((element) {
+    if (shouldFileBeDeleted(element, fileCount)) {
+      element.deleteSync();
+    }
+  });
+}
+
+bool shouldFileBeDeleted(FileSystemEntity file, int fileCount) {
+  var fileStats = file.statSync();
+  final now = DateTime.now();
+  var date = fileStats.modified;
+  if (now.difference(date).inHours >= fileCount) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
Split to service

Reply:
https://github.com/STRDCoders/Ocnera/pull/31/commits/9bce8494e04dca3b02e33e6cbec10a22a2e9f39e